### ISSUE TITLE
image_stylization_transform is broken on windows

### ISSUE THREAD

**Issue by SidShetye** [tensorflow/magenta#1529]
Image stylization code appears brittle and broken on Windows. The jupyter notebook is broken ([reported](https://github.com/tensorflow/magenta-demos/issues/73)) but the underlying model also has bugs (this issue). The model failure results in something like `PermissionError: [Errno 13] Permission denied: 'C:\\Users\\lab\\AppData\\Local\\Temp\\tmpkvttffnd'`. This was also [reported almost 2 years back](https://github.com/tensorflow/magenta/issues/913) (with a simple fix too!) but the issue was ignored and closed. This is still an active bug so reporting it again.

Basically in (now) line 407, of image_utils.py `image = scipy.misc.imread(f.name)` must be fixed to `image = scipy.misc.imread(f)`. 

**But that's only a half-hearted fix**. SciPy documentation [makes it clear](https://docs.scipy.org/doc/scipy/reference/generated/scipy.misc.imread.html) that `scipy.misc.imread` is deprecated and users must switch to `imageio.imread`.

**Comment by bekimd** [comment]
On Ubuntu 18.04
Installed manually magenta-gpu. 
scipy==1.3.1

Traceback (most recent call last):
  File "/home/superuser/pyenv/magenta/bin/arbitrary_image_stylization_with_weights", line 8, in <module>
    sys.exit(console_entry_point())
  File "/home/superuser/pyenv/magenta/lib/python3.6/site-packages/magenta/models/arbitrary_image_stylization/arbitrary_image_stylization_with_weights.py", line 183, in console_entry_point
    tf.app.run(main)
  File "/home/superuser/pyenv/magenta/lib/python3.6/site-packages/tensorflow_core/python/platform/app.py", line 40, in run
    _run(main=main, argv=argv, flags_parser=_parse_flags_tolerate_undef)
  File "/home/superuser/pyenv/magenta/lib/python3.6/site-packages/absl/app.py", line 299, in run
    _run_main(main, args)
  File "/home/superuser/pyenv/magenta/lib/python3.6/site-packages/absl/app.py", line 250, in _run_main
    sys.exit(main(argv))
  File "/home/superuser/pyenv/magenta/lib/python3.6/site-packages/magenta/models/arbitrary_image_stylization/arbitrary_image_stylization_with_weights.py", line 118, in main
    content_img_np = image_utils.load_np_image_uint8(content_img_path)[:, :, :
  File "/home/superuser/pyenv/magenta/lib/python3.6/site-packages/magenta/models/image_stylization/image_utils.py", line 407, in load_np_image_uint8
    image = scipy.misc.imread(f.name)
AttributeError: module 'scipy.misc' has no attribute 'imread'

**Comment by OleksandrGrument** [comment]
It past over 2 year, from the initial post mentioned here. Issue still persist on windows environment. 
Magenta 1.3.0
